### PR TITLE
Scrake antag kit !! (WIP)

### PR DESCRIPTION
**Pull request by yoyobatty**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: YoYoBatty
add: Adds the fucking scrake antag kit! 
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

Well what the fuck is it yoyo, you evil genius ? It's a scrake from the game KILLING FLOOR!! You get this handy dandy uplink kit which gives your evil scrake apron, surgical mask and chainsaw arm! You also walk a little slow and look like a zombie, but you have a ton of armour and health! Don't piss this guy off though or he'll go into a rage sprinting and screaming at you!

Pic of the evil guy: 
image

To be continued...

### CODE REVIEW

**Comment by ghost** [comment]
its wonderful

File: hippiestation/code/game/objects/items/storage/uplink_kits.dm [hunk 1]
```diff
@@ -79,3 +79,14 @@
 /obj/item/storage/box/syndie_kit/armstrong/PopulateContents()
 	new /obj/item/armstrong_scroll(src)
 	new /obj/item/paper/armstrong_tutorial(src)
+
+/obj/item/storage/box/syndie_kit/scrake
+	name = "\improper Brad Armstrong Family Style Karate Kit"
+	desc = "A kit with the necessary tools to become the best karate master on the planet!\
+	Contains a paper letting you know how to fight. \
+	The only cost is your right to not suck at parenting."
```
Comment (ghost) [inline hippiestation/code/game/objects/items/storage/uplink_kits.dm]: you problably seen this but just in case

Comment (yoyobatty) [reply hippiestation/code/game/objects/items/storage/uplink_kits.dm]: yeah, thanks, I just copy pasted the thing to get it to show up ingame

File: hippiestation.dme [hunk 2]
```diff
@@ -2894,6 +2894,7 @@
 #include "hippiestation\code\modules\events\floorcluwne.dm"
 #include "hippiestation\code\modules\events\immovable_rod.dm"
 #include "hippiestation\code\modules\events\holiday\vday.dm"
+#include "hippiestation\code\modules\evil\scrake.dm"
```
Comment (Carbonhell) [inline hippiestation.dme:2897]: Isn't there a mob subfolder for simple mobs?

File: hippiestation/code/modules/evil/scrake.dm [hunk 3]
```diff
@@ -0,0 +1,91 @@
+/obj/item/clothing/suit/apron/chef/scrake
+	name = "scrake's apron"
+	desc = "An apron used to stop getting the gibs of enemies on your skin."
+	allowed = list(/obj/item/tank/internals)
+	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	cold_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
+	min_cold_protection_temperature = SPACE_SUIT_MIN_TEMP_PROTECT
+	clothing_flags = THICKMATERIAL | STOPSPRESSUREDAMAGE
+	armor = list("melee" = 70, "bullet" = 45, "laser" = 80, "energy" = 45, "bomb" = 75, "bio" = 0, "rad" = 30, "fire" = 80, "acid" = 100)
```
Comment (Carbonhell) [inline hippiestation/code/modules/evil/scrake.dm]: Isn't armor a component, does this still work?

File: hippiestation/code/modules/evil/scrake.dm [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+	resistance_flags = INDESTRUCTIBLE | FIRE_PROOF | ACID_PROOF
+	slowdown = 1
+	var/rage_cooldown_duration = 600
+	var/rage_cooldown = 0
+	var/raged = FALSE
+	var/red_splash = list(1,0,0,0.8,0.2,0, 0.8,0,0.2,0.1,0,0)
+	var/pure_red = list(0,0,0,0,0,0,0,0,0,1,0,0)
```
Comment (Carbonhell) [inline hippiestation/code/modules/evil/scrake.dm:16]: What are those two lists supposed to be

File: hippiestation/code/modules/evil/scrake.dm [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+		item_flags = NODROP
+		user.set_species(/datum/species/scrake)
+		user.maxHealth = 250
+		user.health = 250
+
+/obj/item/clothing/suit/apron/chef/scrake/hit_reaction(mob/living/carbon/human/owner, atom/movable/hitby, attack_text = "the attack", final_block_chance = 0, damage = 0, attack_type = MELEE_ATTACK)
+	if(prob(damage*1.25) && !raged)
+		var/mob/living/carbon/human/H = owner
+		Rage(H)
+		spawn(180)
```
Comment (Carbonhell) [inline hippiestation/code/modules/evil/scrake.dm:29]: Timer?

**Review by Carbonhell — CHANGES_REQUESTED** [review]

File: hippiestation/code/modules/evil/scrake.dm (hunk 5 above)
Comment (yoyobatty) [reply hippiestation/code/modules/evil/scrake.dm:29]: timers are gay

**Comment by Carbonhell** [comment]
do we really need yet another boring way to murderbone?

**Comment by Carbonhell** [comment]
I would prefer if this was a Select equipment option (an outfit) for admins to fuck around, rather than yet another traitor murderbone-focused item when we're actually trying to move away from that
-1

**Comment by ghost** [comment]
+1 seems cool

File: hippiestation/code/modules/evil/scrake.dm (hunk 4 above)
Comment (Amelia0010) [reply hippiestation/code/modules/evil/scrake.dm:16]: colour matrices probably

**Comment by earthmoonjune** [comment]
would be better as an adminbus sort of thing rather than a traitor item imo. -1

**Comment by Carbonhell** [comment]
may you change it so that it cannot be bought, and make it an admin select equipment outfit?

**Comment by Carbonhell** [comment]
Closing due to inactivity/denied, but you may reopen it if you do the changes requested.

**Pull request closed without merging**